cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -163,6 +170,7 @@ describe('spec needing owner', () => {
         .findByRole('button', {
           name: 'Save Event Changes',
         })
+        .should('be.enabled')
```

Comment:
Could it be enough to just check if it's enabled?  It's possible that it's failing because Cypress is clicking too quickly, before the button has a chance to become enabled.

Reply:
I dug a little more and initially it looked like that was _unfortunately no_. I've even added check what was the actual value selected at the end. Then an anomaly caught my eye - selected value was `3`. All this time we were being fooled by `.select` method, when passed integer, it will select option by the index, not the value.